src/utils/displayFormatting.ts

```diff
@@ -0,0 +1,11 @@
+// Ref: https://github.com/celo-org/developer-tooling/blob/master/packages/sdk/base/src/displayFormatting.ts#L2
```

Comment:
nit: would be nice to use the immutable link with the sha, so the link doesn't become dead if the file is moved.